cloudflare/app/app.css

```diff
@@ -1,4 +1,4 @@
-@import "tailwindcss";
+@import "tailwindcss" source(".");
```

Comment:
This change is so that the tailwindcss plugin doesn't watch config files in the root of the project, causing additional client reloads. See https://tailwindcss.com/docs/detecting-classes-in-source-files#setting-your-base-path.